src/app/archive/archive_lib/test.ml

```diff
@@ -109,8 +109,9 @@ let%test_module "Archive node unit tests" =
           |> Or_error.ok_exn )
       |> fun _ ->
       let%map (parties : Parties.t) =
-        Mina_generators.Parties_generators.gen_parties_from ~fee_payer_keypair
-          ~keymap ~ledger ~protocol_state_view:genesis_state_view ()
+        Mina_generators.Parties_generators.gen_parties_from ~max_token_parties:0
```

Comment:
Could you create an issue for this and add a TODO here with the issue number? and could you add back the unit test with tokens you wrote in one of the commits until this is fixed?